nomics/README.md

```diff
@@ -337,3 +337,44 @@
   "statusCode": 200
 }
 ```
+
+## Marketcap Endpoint
```

Comment:
Same thoughts as other adapters about just sharing the endpoint using a `path` param